modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -19,12 +19,7 @@ import fs2.io.file.Path
 object method {
 
   sealed trait Run extends CliMethod {
-    val keyStore: StorePath
-    val alias: KeyAlias
-    val password: Password
     val dbConfig: DBConfig
-    val httpConfig: HttpConfig
-    val environment: AppEnvironment
```

Comment:
They are required in `CliMethod` so they don't need to be added here additionally. `RunGenesis` and `RunValidator` will still implement them.